src/test/java/hudson/plugins/junitattachments/AttachmentPublisherTest.java

```diff
@@ -37,15 +37,17 @@ public void testWellKnownFilenamesAreAttached() throws Exception {
         ClassResult cr = getClassResult(action, "test.foo.bar", "DefaultIntegrationTest");
 
         AttachmentTestAction ata = cr.getTestAction(AttachmentTestAction.class);
-        assertNotNull(ata);
+        assert ata != null;
```

Comment:
why replace the jUnit assertions with `assert`?

Reply:
@pjhampton I had some errors in IDEA intellij for assertNotNull, although Junit was enabled..

Reply:
Something wrong with IDEA configuration then.